sample/SampleAspNetCoreApp/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SampleAspNetCoreApp.Controllers
+{
+	/**
```

Comment:
Just a small nit: we use [Xml doc](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/) format here in the repo, like [here](https://github.com/elastic/apm-agent-dotnet/blob/master/src/Elastic.Apm/Api/ITracer.cs#L8) - In Rider or Visual Studio just hit `/` 3 times and the IDE will generate it for you.

Reply:
OK. got it.